tide/master/MasterApplication.h

```diff
@@ -80,6 +81,7 @@ class MasterApplication : public QApplication
     virtual ~MasterApplication();
 
 private:
+    LoggingUtility _logger;
```

Comment:
For consistency, please make this a unique_ptr<LogginUtility>, move it down below the _restInterface declatation, and move #include "LoggingUtility.h" to the .cpp